cc/private/rules_impl/native.bzl

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 # Redefine native symbols with a new name as a workaround for
-# exporting them in `//third_party/bazel_rules/rules_proto/proto:defs.bzl` with their original name.
+# exporting them in `//third_party/bazel_rules/rules_cc/cc:defs.bzl` with their original name.
```

Comment:
Does this need a copybara config change?

Reply:
You mean internal one? No, it doesn't seem so.